frontend/src/components/VSafeBrowsing/VSafeBrowsing.vue

```diff
@@ -65,12 +66,14 @@ export default defineComponent({
     const sensitivityPath = computed(() => app.localePath("/about")) // TODO: Issue#2550
 
     const featureFlagStore = useFeatureFlagStore()
+    const { sendCustomEvent } = useAnalytics()
 
     let fetchSensitive = computed(() =>
       featureFlagStore.isOn("fetch_sensitive")
     )
     let setFetchSensitive = ({ checked }: { checked: boolean }) => {
       featureFlagStore.toggleFeature("fetch_sensitive", checked ? ON : OFF)
+      sendCustomEvent("TOGGLE_FETCH_SENSITIVE", { checked })
```

Comment:
`checked` can be `undefined` as well. This change was added in #2897.

Reply:
I don't know what needs to be changed here. Type check is passing.